src/Domain/FunctionsOfTime/OptionTags.hpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+#include "Options/Auto.hpp"
+#include "Options/Options.hpp"
+
+namespace domain::FunctionsOfTime::OptionTags {
+/*!
+ * \ingroup OptionGroupsGroup
+ * \brief Groups options for reading in FunctionOfTime data from SpEC
+ */
+struct CubicFunctionOfTimeOverride {
+  static constexpr Options::String help{
+      "Options for importing FunctionOfTimes from SpEC"};
+};
+
+/*!
+ * \brief Path to an H5 file containing SpEC FunctionOfTime data
+ */
+struct FunctionOfTimeFile {
+  using type = Options::Auto<std::string>;
```

Comment:
why is this Auto?

Reply:
It shouldn't be anymore...thanks for catching this! This is leftover from when I was avoiding adding this option to every executable's input files by making it Auto. But now, the option only gets added if the executable sets a compile-time flag that you'll be overriding the functions of time, so making the option Auto is no longer necessary. I am changing it to not be Auto in my fixup.